parse/grammar/KuneiformParser.g4

```diff
@@ -177,12 +203,19 @@ procedure_return:
     The following section includes parser rules for SQL.
 */
 
-// sql is a top-level SQL statement.
-sql:
-    sql_statement SCOL
+// sql is a top-level SQL statement, it maps to SQLStmt interface in AST.
+sql_stmt:
```

Comment:
To change `sql_statement` to `sql_stmt` in action/procedure block will break the parser/analyzer/planner, better do it together when we make the changes

Reply:
Agreed